packages/tables/src/Table.php

```diff
@@ -339,6 +339,23 @@ public function getRecordUrl(Model $record): ?string
         return $callback($record);
     }
 
+    public function isRecordBulkActionEnabled(Model $record): bool {
+        if (!$this->isSelectionEnabled()) {
+            return false;
+        }
+
+        /** @var TableComponent $livewire */
+        $livewire = $this->getLivewire();
+
+        $callback = invade($livewire)->getTableRecordBulkActionsEnabled();
+
+        if (! $callback) {
+            return true;
+        }
+
+        return $callback($record) ?? true;
```

Comment:
Could be reduced to:

```suggestion
        return $callback ? $callback($record) : true;
```